### PR TITLE
regex take two

### PR DESCRIPTION
**Pull request by rachelg4**
@stvstnfrd

### CODE REVIEW

File: playbooks/roles/logstash/templates/logstash.conf.j2 [hunk 1]
```diff
@@ -45,13 +45,11 @@ filter {
     grok {
       match => { "message" => "%{GREEDYDATA:@message}" }
     }
-  } else if [type] == "supervisor_worker_lms_error" {
+  } else if [type] == "supervisor_worker_error" {
     grok {
       match => { "message" => "\[%{TIMESTAMP_ISO8601:syslog_timestamp}\: %{WORD:log_level}\/%{USERNAME:supervisor_function}\] %{GREEDYDATA:@message}" }
-    }
-  } else if [type] == "supervisor_worker_cms_error" {
```
Comment (stvstnfrd) [inline playbooks/roles/logstash/templates/logstash.conf.j2]: So we no longer need to differentiate between lms and cms here?

Comment (rachelg4) [inline playbooks/roles/logstash/templates/logstash.conf.j2]: correct- they both have both

File: playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 2]
```diff
@@ -1,19 +1,13 @@
 filebeat:
-  prospectors:
+ prospectors:
```
Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: nuke

File: playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 3]
```diff
@@ -25,55 +19,59 @@ filebeat:
...
+        - "{{ FILEBEAT_LOG_DIR }}/supervisor/cms_*-stderr.log"
         - "{{ FILEBEAT_LOG_DIR }}/supervisor/lms_*-stderr.log"
-      document_type: supervisor_worker_lms_error
+      document_type: supervisor_worker_error
       multiline:
-        pattern: '^\[[0-9][0-9][0-9][0-9]'
+        pattern: '^[[:digit:]]{4}|^\[[[:digit:]]{4}'
+        negate: true
         match: after
+        timeout: 10s
```
Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: Parameterize this timeout (everywhere).

File: playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 4]
```diff
@@ -25,55 +19,59 @@ filebeat:
...
     -
       paths:
         - "{{ FILEBEAT_LOG_DIR }}/devpi/supervisor/supervisord.log"
         - "{{ FILEBEAT_LOG_DIR }}/devpi/supervisor/devpi-server-stderr.log"
       document_type: devpi
-      multiline:
-        pattern: '^([0-9][0-9][0-9][0-9]|\[[0-9][0-9][0-9][0-9])'
-        match: after
     -
       # these were not present in the sandbox
```
Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: Update this (and related) comment to match new reality.

File: playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 5]
```diff
@@ -25,55 +19,59 @@ filebeat:
...
     -
-      # These do not have time stamps, so their multiline concatenation
-      # is handled differently.
       paths:
         - "{{ FILEBEAT_LOG_DIR }}/supervisor/xqueue_consumer-stderr.log"
         - "{{ FILEBEAT_LOG_DIR }}/supervisor/certs-stderr.log"
       document_type: supervisor_error_traceonly
       multiline:
-        pattern: '^Traceback'
+        pattern: '^[[:upper:]][[:lower:]]{8}'
```
Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: If at all possible, let's match this (sub)string explicitly.

File: playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 6]
```diff
@@ -84,39 +82,40 @@ filebeat:
...
+        negate: true
         match: after
+        timeout: 10s
     -
       paths:
         - "{{ FILEBEAT_LOG_DIR }}/supervisor/forum-stderr.log"
       document_type: supervisor_forum_error
       multiline:
-        pattern: '^(I\, \[|[0-9][0-9][0-9]\.)'
+        pattern: '^(I, \[|[[:digit:]]{3}.)'
```
Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: Period?

Comment (stvstnfrd) [inline playbooks/roles/filebeat/templates/filebeat.yml.j2]: Find a more explicit way to match; there are too many potential false positives here; consider named groups, if available.

**Pull request merged**